src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -876,6 +876,8 @@ private async Task EntityMiddleware(DispatchMiddlewareContext dispatchContext, F
 
             try
             {
+                entityShim.AddTraceFlag('1');
+

```

Comment:
what does `TraceFlag('1')` do? Can we add a small inline comment clarifying this?

Reply:
Sure. TraceFlags are essentially a "log inside the log". They create a string of log characters. This is because it can be tricky to figure out what path was taken through the code.

Reply:
I like, I can see it being really useful.